lib/ctblgrp.gi

```diff
@@ -2056,11 +2056,16 @@ end );
 ##
 #F  DixonSplit(<D>) . .  calculate matrix,split spaces and obtain characters
 ##
-InstallGlobalFunction( DixonSplit, function(D)
-local r,i,j,ch,ra,bsm,
+InstallGlobalFunction( DixonSplit, function(arg)
```

Comment:
This is never called with the new argument, though, right? So is this for future work, or just for manual invocations by "super expert users"?

Reply:
The latter.